### DISCUSSION TITLE
Rebase local branch onto remote-tracking branch

### DISCUSSION THREAD

**Discussion by akhayyat** [jesseduffield/lazygit#1947] [Q&A]
How can I, using lazygit, rebase a local branch (e.g. `feature-1`) on its remote-tracking branch (e.g. `origin/feature-1`) in case there has been some divergent changes (new commits on `origin/feature-1` that are not on `feature-1` and vice versa)?

**Comment by tg0h** [comment]
<img width="1325" alt="image" src="https://user-images.githubusercontent.com/7954426/170179953-d3b3dcae-2dc3-4ad0-8820-ff8bc45fa0d9.png">

try this, it should work

**Reply by laurentS** [reply]
I just tried this, it works as long as you first fetch the remote branch :sweat_smile:
